plugins/services/src/js/components/ConfigurationMapEditAction.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import ConfigurationMapAction from '../../../../../src/js/components/ConfigurationMapAction';
+
+const ConfigurationMapEditAction = ({tabViewID, handleEditClick}) => {
+  if (!handleEditClick) {
+    return null;
```

Comment:
I think this is the cause of the error when viewing a deployed service's config; we have to return `<noscript />` from stateless functional components.